### ISSUE TITLE
Yoga options does not accept schema or yoga property

### ISSUE THREAD

**Issue by arkmech** [elysiajs/elysia-graphql-yoga#3]
Example from: https://www.npmjs.com/package/@elysiajs/graphql-yoga
```ts
import { Elysia } from 'elysia'
import { yoga } from '@elysiajs/graphql-yoga'

import { createYoga, createSchema } from 'graphql-yoga'

const app = new Elysia()
    .use(
        yoga({
           // yoga property doesn't exist
            yoga: createYoga({
                schema: createSchema({
                    typeDefs: `
                        type Query {
                            hi: String
                        }
                `,
                resolvers: {
                    Query: {
                        hi: () => 'Hi from Elysia'
                    }
                }
            })
        })
    )
    .listen(8080)
```
Error
```
Object literal may only specify known properties, and 'yoga' does not exist in type 'ElysiaYogaConfig<string, MaybePromise<Record<string, unknown>> | ((initialContext: YogaInitialContext) => unknown) | undefined>'
```

Desired, yoga accepts schema property `createSchema()` or from `makeExecutableSchema()`, can only do `typeDefs` and `resolvers` separately.
```ts
  app.use(yoga({ schema }));
```

**Comment by fecony** [comment]
Example provided in npm registry and this repo readme is outdated from previous version
You can read how to use plugin on the docs plugin page as they are updated https://elysiajs.com/plugins/graphql-yoga.html

**Comment by fecony** [comment]
@SaltyAom How about sending users to the documentation page instead of putting the same code in the readme.md file of the plugin repo? That way, the documentation would become the source of truth instead of trying to keep it up to date with the documentation and vice versa

**Comment by arkmech** [comment]
@Fecony , got it. Thanks!

Regardless, would be nice if this was available, for those that do schema stitching
```ts
  app.use(yoga({ schema }));
```

**Comment by fecony** [comment]
> Regardless, would be nice if this was available, for those that do schema stitching

Yes agreed, I wanted to use https://garph.dev/ once but Elysia plugin doesn't support passing schema. Maybe it would be possible to adjust plugin to let us pass schema or what we do now

But I didn't try to do it and see how hard it would be, pretty sure @SaltyAom can do it while sleeping xd

**Comment by arkmech** [comment]
> > Regardless, would be nice if this was available, for those that do schema stitching
> 
> Yes agreed, I wanted to use https://garph.dev/ once but Elysia plugin doesn't support passing schema. Maybe it would be possible to adjust plugin to let us pass schema or what we do now
> 
> But I didn't try to do it and see how hard it would be, pretty sure @SaltyAom can do it while sleeping xd

oh wow...that looks enticing. I might try garph out

**Comment by SaltyAom** [comment]
Just wake up, make `schema` property to accept `TypeSource`.

Keep in mind that you will not have type-safety from Mobius if you are using external source.

**Comment by arkmech** [comment]
Just upgraded to `0.6.3`, and don't see the `schema` key

**Comment by nowlena** [comment]
Agreed, I'd love to be allowed to pass in our own "schema".

For reference, it's because i want to use Pothos (https://github.com/hayes/pothos)

**Comment by arkmech** [comment]
I'm using `graphql-middleware` as well. And their `applyMiddleware()` returns `GraphQLSchemaWithFragmentReplacements` which is compatible with `createYoga()` schema type. Don't think there is a way to pull out the typeDefs and resolvers separately from a schema.

```ts
schema?: YogaSchemaDefinition<{}> | undefined
```

If there is anything I can do to help.

**Comment by G9000** [comment]
any plan for this yet? using `graphql-tools/merge `works but all the type gone 🥲

**Comment by renhyl** [comment]
Pull Request: https://github.com/elysiajs/elysia-graphql-yoga/pull/8
